lib/app_state/lazy_load_success.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:fluffychat/app_state/success.dart';
+
+abstract class LazyLoadSuccess<T> extends Success {
+  final List<T> data;
+  final int offset;
+  final bool isEnd;
```

Comment:
Do all success states in the app have it?

Reply:
no, only the `resource` which have `fetch` or `search` supports `load more`.

Reply:
So I think I should give this class a different name. avoid confusion later. :))

Reply:
@dab246 I think LazyLoadSuccess is enough, it support for a list with lazy load feature

Reply:
ok for me `LazyLoadSuccess`